src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onSelected }:
+{ tabs: Tab[],
+  selectedTabId: string,
+  onSelected: (tabTitle: string) => void }) => {
```

Comment:
```suggestion
export const Tabs: React.FC<Props> = ({ 
  tabs, 
  selectedTabId, 
  onSelected,
}) => {
```
Creat an interface for Props